config/tips.txt

```diff
@@ -1,92 +1,155 @@
-Locked down rogue borgs can be dismantled to return the MMI for placement in a new borg.
-Fore = north, aft = south, starboard = east and port = west.
-Many things that bind you can be resisted out of with the resist button. This includes locked or welded lockers, chairs and handcuffs. Whenever stuck, just give it a try!
-You can use . or # instead of : for radio channels. Building this habit reduces the chance of accidentally saying things on common.
-While pulling something, clicking on a tile with an empty hand will move it to that tile. No more getting stuck in maint while dragging something!
-Things that override clicks like decks of cards or paper bins can still be picked up by dragging them to your character. Equipped things that override click can be unequipped by dragging it to an empty hand.
-In the job selection menu, from the lobby, you can use right click on the "Low/Medium/High" priority button to make it cycle backwards.
-If you stay out of camera range for 10 seconds, the AI's track on you will break.
-You can shoot cameras with bullets to disable them remotely.
-Disabling a camera with wirecutters will not cause a camera alarm, but bludgeoning it will.
-Gangs derive their power from their presence and notoriety. Cleaning up gang tags is the best way to slow their growth.
+Glass shards can be welded to make glass, and metal rods can be welded to make metal. Ores can be welded too, but this takes a lot of fuel.
+If you need to drag multiple people either to safety or to space, bring a locker over and stuff them all in before hauling them off.
+You can grab someone by clicking on them with the grab intent, then upgrade the grab by clicking on the grab button in your active hand. An aggressive grab will allow you to place someone on a table by clicking on it, or throw them by toggling on throwing.
+You can destroy computers by shooting lasers at them.
+Holding alt and left clicking a tile will allow you to see its contents in the top right window pane, which is much faster than right clicking.
+The resist button will allow you to resist out of handcuffs, being buckled to a chair or bed, and out of locked lockers. Whenever you're stuck, try resisting!
+You can move an item out of the way by dragging it and then clicking on an adjacent tile with an empty hand.
+You can recolor certain items like jumpsuits and gloves in washing machines by also throwing in a crayon.
+Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using.
+As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your pants or your life are things to worry about.
+As the Captain, always take the nuclear disk and pinpointer with you every shift. If you don't trust yourself, or you trust the Head of Security sufficiently, give either one or both to him.

```

Comment:
Well I mean just in case only the disc is stolen, but yeah I see your point. I'll rephrase that to be more suggestive of giving either one to somebody else.